assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -230,6 +230,29 @@ function APIProvider({ children }) {
     return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
   }, [users]);
 
+  const saveMetaBoxes = useCallback(
```

Comment:
This callback is inspired by this section in Gutenberg: https://github.com/WordPress/gutenberg/blob/148e2b28d4cdd4465c4fe68d97fcee154a6b209a/packages/edit-post/src/store/effects.js#L72-L126

Reply:
This is worth a comment in the code as well.